recipes/tuiview/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.1.8" %}
+
+package:
+    name: tuiview
+    version: {{ version }}
+
+source:
+    fn: TuiView-{{ version }}.tar.gz  # [unix]
+    fn: TuiView-{{ version }}.zip  # [win]
+    url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-1.1.8.tar.gz  # [unix]
+    url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-1.1.8.zip  # [win]
```

Comment:
Problem was that the .zip file (created using sdist on Windows) contains the .bat files, whereas the .tar.gz (from Linux) does not. See https://bitbucket.org/chchrsc/tuiview/src/1a17bc21d8064ce5510d654efb9e1400a5140709/setup.py?fileviewer=file-view-default#setup.py-111. 
Maybe I should just be using the .zip file for both platforms?
Is supplying a .bat file so TuiView can be easily started from the command line still Conda practice?